packages/cli/test/unit/commands/env.test.ts

```diff
@@ -183,6 +183,9 @@ describe('env', () => {
       );
 
       await expect(pullPromise).resolves.toEqual(0);
+
+      client.setArgv('env', 'rm', 'NEW_VAR', '--yes', '--cwd', cwd);
+      await env(client);
```

Comment:
Can we put this in a `finally` block to make sure the cleanup always happens when the test fails?